evals/eval.py

```diff
@@ -153,3 +158,11 @@ def worker_thread(args):
                 iter = pool.imap_unordered(worker_thread, work_items)
             idx_and_result = list(tqdm(iter, total=len(work_items), disable=not show_progress))
         return [r for _, r in sorted(idx_and_result)]
+
+    def get_samples(self):
```

Comment:
Could we separate out this change from the PR? Currently all evals are using samples_jsonl, but I'd like to add support for other ways of loading samples, and having this in the Eval base class might be confusing.

Reply:
I think one more motivation for this is dynamic downloading of an eval which is stored via `git-lfs`, we were planning on updating this function where, if the samples_jsonl is an lfs file, fetch and download it.

Reply:
Oh I see! Let's keep it in then.